src/components/v2/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+
+import Typography from '@mui/material/Typography';
+import {
+  BASE_BSC_SCAN_URL,
+  VENUS_MEDIUM_URL,
+  VENUS_DISCORD_URL,
+  VENUS_TWITTER_URL,
+  VENUS_GITHUB_URL,
+} from 'config';
+import { Icon } from 'components/v2/Icon';
+import { useStyles } from './styles';
+
+export interface IFooterProps {
+  currentBlockNumber: number;
+}
+
+export const Footer: React.FC<IFooterProps> = ({ currentBlockNumber }) => {
+  const styles = useStyles();
+
+  return (
+    <div css={styles.Container}>
+      <Typography
```

Comment:
It would be nice to have these setup as components and not use Typography directly. For example, all our a tags will likely have similar props

```
target="_blank"
rel="noreferrer"
component="a"
variant="small2"

Reply:
They most probably won't actually, `small2` isn't a common variant for texts.
I don't think there's any advantage in making a separate component for this, it wouldn't be reused enough times to justify that.